README.md

```diff
@@ -2,57 +2,79 @@
 ![Version](https://badge.fury.io/py/lumigo-tracer.svg)
 ![codecov](https://codecov.io/gh/lumigo-io/python_tracer/branch/master/graph/badge.svg?token=6EgXIlefwG)
 
+This is the Python version of Lumigo's tracer library.
+
 Supported Runtimes: 3.6, 3.7, 3.8
 
-# How To Use
-## With Lambda Layers:
-* Use the latest ARN version [from these tables](https://github.com/lumigo-io/python_tracer/tree/master/layers)
-## With Serverless framework:
-* Install the [**serverless-lumigo-plugin**](https://github.com/lumigo-io/serverless-lumigo-plugin/blob/master/README.md)
-## Manually
-* install with `pip install lumigo_tracer` <br/> 
-* import using `from lumigo_tracer import lumigo_tracer`
-* wrap you lambda function using `@lumigo_tracer` or `@lumigo_tracer(token='XXX')`. As an example, your lambda should look like: 
-```
+# Using the tracer
+There are three ways to use the tracer with your AWS Lambda functions: using our Lambda Layer, using the Serverless framework, or instrumenting your functions manually.
```

Comment:
@MattBillock the structure and wording of this readme should be identical to nodejs tracer, in the end they both offer similar capabilities the main difference is the runtime they support. I don't mind using this wording or the nodejs wording, both seem ok, but make them identical.

They do differ in implementation details and the examples, but all in all they are very similar

Reply:
@efimk-lu - thanks for the feedback, I made adjustments so that the text more closely mirrors the lumigo-node package. Let me know what you think of the new version